components/automate-cli/cmd/chef-automate/automateHADeployExistingInfra.go

```diff
@@ -247,24 +247,15 @@ func (e *existingInfra) validateExternalDbFields() *list.List {
 	return errorList
 }
 
-func extractIPsFromCertsByIP(certsByIp []struct {
-	IP         string `toml:"ip"`
-	PrivateKey string `toml:"private_key"`
-	PublicKey  string `toml:"public_key"`
-}) []string {
+func extractIPsFromCertsByIP(certsByIp []CertByIP) []string {
 	ips := []string{}
 	for _, el := range certsByIp {
 		ips = append(ips, el.IP)
 	}
 	return ips
 }
 
-func extractIPsFromCertsByIPOpensearch(certsByIp []struct {
-	IP         string `toml:"ip"`
-	PrivateKey string `toml:"private_key"`
-	PublicKey  string `toml:"public_key"`
-	NodesDn    string `toml:"nodes_dn"`
-}) []string {
+func extractIPsFromCertsByIPOpensearch(certsByIp []CertByIP) []string {
```

Comment:
Both extractIPsFromCertsByIPOpensearch and extractIPsFromCertsByIP are similar. Can we replace one with another?